guides/doc-Provisioning_Guide/topics/Networking.adoc

```diff
@@ -376,14 +389,14 @@ if option architecture = 00:06 {
 } elsif option architecture = 00:09 {
   filename "grub2/shim.efi";
 } elsif exists user-class and option user-class = "iPXE" {
-  filename "http://_satellite.example.com_/unattended/iPXE";
+  filename "http://_{foreman.example.com}_/unattended/iPXE";
 } else {
   filename "pxelinux.0";
 }
 ----
 +
 [NOTE]
-For `http://satellite.example.com/unattended/iPXE`, you can also use a {ProjectName} {SmartProxy} `http://capsule.example.com:8000/unattended/iPXE`.
+For `http://{foreman.example.com}/unattended/iPXE`, you can also use a {ProjectName} {SmartProxy} `http://{smartproxy.example.com}:8000/unattended/iPXE`.
```

Comment:
Is this the same port issue @ekohl ? Katello is 8000 but Foreman is I think 8448? Or maybe not.

Reply:
Foreman uses port 8000 but disables HTTP by default so there's nothing actually listening. You will need to pass in `--foreman-proxy-http true` to the installer.

Reply:
Please update the note then:

Use `http://{foreman.example.com}/unattended/iPXE` when {SmartProxy} HTTP endpoint is disabled (installer option `--foreman-proxy-http false`). Template {SmartProxy} plugin has a default value 8000 when enabled and can be changed with `--foreman-proxy-http-port` installer option, in that case use `http://{smartproxy.example.com}:8000`./unattended/iPXE` when using

The port number can be changed via `--foreman-proxy-http-port` option or even disabled via  which requires

Reply:
this bit makes no sense @lzap "in that case use"

Reply:
There are a few problems

remove "endpoint" or find another word? We don't use that. 

Use `http://{foreman.example.com}/unattended/iPXE` when {SmartProxy} HTTP the installer option `--foreman-proxy-http` is set to `false`. The template {SmartProxy} plugin has a default value of `8000` when enabled and can be changed with the  `--foreman-proxy-http-port installer` option. 

And I don't understand the last part. "If you change the port, use x" ?

Reply:
Sure. 

> And I don't understand the last part.

Let's drop it then.